### PR TITLE
Contacts

### PR DESCRIPTION
**Pull request by Johntosh**
contact us model

### CODE REVIEW

File: config.py
Comment (DennisRono) [inline config.py:1]: there are changes to configuration file that are way ahead of this file please pull from dev first

File: order_product.py
Comment (DennisRono) [inline order_product.py:1]: this file will introduce an error setifattr is supposed to be setattr

**Review by DennisRono — CHANGES_REQUESTED** [review]
Pull for latest changes and fix errors commented

**Review by Kigo-P — CHANGES_REQUESTED** [review]
Conflicts arose because you do not have the latest changes

**Pull request closed without merging**